src/python/aqueduct_executor/operators/utils/utils.py

```diff
@@ -164,13 +169,26 @@ def _write_json_output(
     storage.put(output_path, json.dumps(output).encode(_DEFAULT_ENCODING))
 
 
+def _write_none_output(
+    storage: Storage,
+    output_path: str,
+    output: Any,
+) -> None:
+    # default to write a empty bytes
+    if output is None:
```

Comment:
The assumption here is that output is None already for our inference, so we shouldn't even check it. Can you just remove `output` completely and just have this be a single `storage.put()` line like the others?